src/samples/GroupChannelUpdateDeleteMessageByOperator.js

```diff
@@ -0,0 +1,658 @@
+
+}
+
+const ChannelHeader = ({ children }) => {
+    return <div className="channel-header">{children}</div>;
+
+}
+
+const MembersList = ({ members, handleMemberInvite, registerUnregisterAnOperator, userIsOperator, userIdInputValue }) => {
+    if (members) {
+        return <div className="members-list">
+            <button onClick={handleMemberInvite}>Invite</button>
+            {members.map((member) => {
+              const isOperator = (member.role === "operator");
+              const memberIsSender = (member.userId !== userIdInputValue);
+              return(
+                <div key={member.userId}>
+                  {userIsOperator && <div key={member.userId} className="member-item-wrapper">
+                    <div className="member-item">
+                      {member.nickname}
+                      {isOperator && <img className="message-icon" src='/operator_icon.png' />}
+                    </div>
+                    {memberIsSender && <button onClick={() => registerUnregisterAnOperator(member)}>
+                      {isOperator ? "Unregister as operator" : "Register as operator"}
+                    </button>}
+                  </div>}
+                  {!userIsOperator && <div className="member-item">{member.nickname}</div>}
+                </div>
+              )
+            })}
+        </div>;
+    } else {
+        return null;
+    }
+
+
+}
+
+const MessagesList = ({ messages, handleDeleteMessage, updateMessage, currentlyJoinedChannel }) => {
+    return <div className="message-list">
+        {messages.map(message => {
+            const messageSentByYou = message.sender.userId === sb.currentUser.userId;
+
+            return (
+                <div key={message.messageId} className={`message-item ${messageSentByYou ? 'message-from-you' : ''}`}>
+                    <Message
+                        message={message}
+                        handleDeleteMessage={handleDeleteMessage}
+                        updateMessage={updateMessage}
+                        currentlyJoinedChannel={currentlyJoinedChannel}
+                        messageSentByYou={messageSentByYou} />
+                    <ProfileImage user={message.sender} />
+
+                </div>);
+        })}
+    </div >
+}
+
+const Message = ({ message, updateMessage, handleDeleteMessage, messageSentByYou, currentlyJoinedChannel }) => {
+    if (message.url) {
+        return (
+            <div className={`message  ${messageSentByYou ? 'message-from-you' : ''}`}>
+                <div className="message-user-info">
+                    <div className="message-sender-name">{message.sender.nickname}{' '}</div>
+                    <div>{timestampToTime(message.createdAt)}</div>
+                </div>
+                <img src={message.url} />
+            </div >);
+    }
+    const messageSentByCurrentUser = message.sender.userId === sb.currentUser.userId || currentlyJoinedChannel.myRole === 'operator';
```

Comment:
think the variable name makes less sence now.

```suggestion
    const showAdditionalMessageControls = message.sender.userId === sb.currentUser.userId || currentlyJoinedChannel.myRole === 'operator';
```